tools/argument-explanation.md

```diff
@@ -0,0 +1,203 @@
+        - If not, continue to the next step.
+
+    2. If `ignore tables` is not empty, judge whether a matched rule exists in `ignore-tables`.
+    
+        - If yes, ignore it and exit.
+        - If not, continue to the next step.
+
+    3. If `do-tables` is not empty, ignore it and exit. Otherwise, exit and execute the statement.
+
+## Filter rules of binlog events
+
+```
+# table level
+user-filter-1:
+​    schema-pattern: "test_*"     # `schema-pattern`/`table-pattern` uses the wildcard matching rule.
+​    table-pattern: "t_*"
+​    events: ["truncate table", "drop table"]
+​    sql-pattern: ["^DROP\\s+PROCEDURE", "^CREATE\\s+PROCEDURE"]
+​    action: Ignore
+
+# schema level
+user-filter-2:
+​    schema-pattern: "test_*"
+​    events: ["All DML"]
+​    action: Do
+```
+
+For the matching rules, see
+Description: Configures the filter rules of black white list for binlog events of the upstream tables and DDL SQL statements that match `schema-pattern`/`table-pattern`.
+
+- `events`: the binlog event array
+
+    | Events            | Type | Description                   |
+    | ---------------   | ---- | ----------------------------- |
+    | `all`             |      | Includes all the events below |
+    | `all dml`         |      | Includes all DML events below |
+    | `all ddl`         |      | Includes all DML events below |
+    | `none`            |      | Includes none of the events below |
+    | `none ddl`        |      | Includes none of the DDL events below |
+    | `none dml`        |      | Includes none of the DML events below |
+    | `insert`          | DML  | Inserts a DML statement              |
+    | `update`          | DML  | Updates a DML statement              |
+    | `delete`          | DML  | Deletes a DML statement              |
+    | `create database` | DDL  | Creates a database         |
+    | `drop database`   | DDL  | Drops a database           |
+    | `create table`    | DDL  | Creates a table      |
+    | `create index`    | DDL  | Creates an index          |
+    | `drop table`      | DDL  | Drops a table              |
+    | `truncate table`  | DDL  | Truncates a table          |
+    | `rename table`    | DDL  | Renames a table            |
+    | `drop index`      | DDL  | Drops an index           |
+    | `alter table`     | DDL  | Alters a table           |
+
+- `sql-pattern` 
+
+    - Filters a specific DDL SQL statement. 
+    - The matching rule supports using an regular expression, for example, `"^DROP\\s+PROCEDURE"`.
+    - **Note:** If `sql-pattern` is empty, no filtering operation is performed. For the filter rules, see the `action` description.
+
+- `action` 
+
+    - String (`Do`/`Ignore`)
+    - For rules that match `schema-pattern`/`table-pattern`, judge whether the DDL statement is in the events of the rule or `sql-pattern`:
+        
+        - If the DDL statement is in the events of the rule or `sql-pattern`, execute `Ignore` (black list).
```

Comment:
`Do` and `Ignore` are options for `action`.
I think keeping the format like CN doc is better

Reply:
@lilin90 Please pay special attention to this.